lib/broadside/configuration.rb

```diff
@@ -36,9 +38,36 @@ def ecs
       @ecs ||= EcsConfig.new
     end
 
+    def ssh_cmd(ip, options = { tty: false })
+      cmd = 'ssh -o StrictHostKeyChecking=no'
+      cmd << ' -t -t' if options[:tty]
+      cmd << " -i #{@ssh[:keyfile]}" if @ssh[:keyfile]
+      if @ssh[:proxy]
+        cmd << " -o ProxyCommand=\"ssh #{@ssh[:proxy][:host]} nc #{ip} #{@ssh[:proxy][:port]}\""
+      end
+      cmd << " #{@ssh[:user]}@#{ip}"
```

Comment:
maybe you should `raise ArgumentError` on missing `ssh[:user]`? looks essential to building a functional command